docs/development_suite/identity-and-access-management/console-levels-and-permission-management.md

```diff
@@ -3,25 +3,26 @@ id: console-levels-and-permission-management
 title:  Console levels and permission management
 sidebar_label: Console Levels and Permission Management
 ---
-Assigning different Roles and permissions to each identity in the Console is a key action for defining responsibilities within your platform ecosystem.
-The Console is based on hierarchical levels, and for each of them, specific permissions and capabilities have been identified and assigned to Users and Service Accounts.
+Assigning different roles and permissions to each [Identity](/development_suite/identity-and-access-management/overview.md#identity-and-access-management) that has access to Mia-Platform Console is a key action for defining responsibilities within your platform ecosystem.
+The Console is based on hierarchical levels and, for each of them, specific permissions and capabilities have been identified and can be assigned.  
 Let's see how they are configured.
 
 ## Console Levels
 
 The Console resources are organized in a hierarchical structure on three levels:
 
-1. **Console level** is the highest level and allows the general configuration of your Console. At this level you can configure the Companies, the different templates for Project creation and you have control of your Projects and the Marketplace;
+1. **Console level** is the root level regarding the general configuration of your Console. 
+At this level you can configure resources such as Companies, Project Templates and Marketplace.
 
-1. **Company level** is the second hierarchical level. A Company can contain several underlying Projects, which can inherit different kinds of information from the Company without needing further configuration.
+1. **Company level** is the second hierarchical level. A Company can contain several underlying Projects, which can inherit different kinds of information from the Company without needing further configuration. At this level, License, Clusters, Providers, Users, Service Account Users, Groups can be managed. To know more about Company configuration, check out the dedicated [documentation section](/development_suite/company/create.mdx).
```

Comment:
```suggestion
1. **Company level** is the second hierarchical level. A Company can contain several underlying Projects, which can inherit different kinds of information from the Company without needing further configuration. At this level, License, Clusters, Providers, Users and Service Accounts can be managed. To know more about Company configuration, check out the dedicated [documentation section](/development_suite/company/create.mdx).
```

I'd add groups reference in PR #125 once this one is merged. Without Groups references we can bring these changes to production before the release of groups management

Reply:
clear, thanks for the review